### PR TITLE
docs: FAQ fixes and some info about snap sync

### PR DESCRIPTION
**Pull request by AvivYaish**
Some minor spelling and grammar changes.

### CODE REVIEW

File: docs/_interface/FAQ.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ sort_key: C
 
 ---
 
-**Q.** How do Ethereum syncing work?
+**Q.** How does Ethereum syncing work?
 
-**A.** The current default mode of sync for Geth is called fast sync. Instead of starting from the genesis block and reprocessing all the transactions that ever occurred (which could take weeks), fast sync downloads the blocks, and only verifies the associated proof-of-works. Downloading all the blocks is a straightforward and fast procedure and will relatively quickly reassemble the entire chain.
+**A.** The current default syncing mode used by Geth is called "fast sync". Instead of starting from the genesis block and processing all the transactions that ever occurred (which could take weeks), fast sync downloads the blocks, and only verifies the associated proof-of-works. Downloading all the blocks is a straightforward and fast procedure and will relatively quickly reassemble the entire chain.
```
Comment (s1na) [inline docs/_interface/FAQ.md:21]: We can also update "fast sync" to say "snap sync"

Comment (AvivYaish) [reply docs/_interface/FAQ.md]: I addressed snap sync in [a new commit](https://github.com/ethereum/go-ethereum/pull/24812/commits/c90e90fe438e4742943d35d030478174b930b114).

**Comment by s1na** [comment]
The other benefit being that we can now easily add a ToC

**Comment by s1na** [comment]
I kept most of the fast sync info until we can replace them with some FAQs about snap sync (like different phases, most common problems faced, etc.)

**Pull request merged**